workers/generic-worker/main.go

```diff
@@ -691,6 +696,20 @@ func (task *TaskRun) validateJSON(input []byte, schema string) *CommandExecution
 	return MalformedPayloadError(fmt.Errorf("Validation of payload failed for task %v", task.TaskID))
 }
 
+func validateGenericWorkerBinary() error {
```

Comment:
Can you add a doc comment here to explain what the function does? It isn't totally clear just from the function name. Thanks!